frontend/service/wikibase.service.js

```diff
@@ -23,7 +23,7 @@ export class WikibaseService {
     })
     this.$store = store
     this.$query = new QueryService(store, this.$config)
-    this.$oauth = new OAuthService(store, this.$config)
+    this.$oauth = new OAuthService(store, this.$config, app)
```

Comment:
I propose that remove _this.$config_ parameter from the constructor, in fact, [$this.config=app.$config](https://github.com/PhiloBiblon/philobiblon-ui/blob/master/frontend/service/wikibase.service.js#L16)